Sources/Basic/Path.swift

```diff
@@ -101,12 +101,13 @@ public struct AbsolutePath: Hashable {
     /// Convenience initializer that verifies that the path is absolute.
     public init(validating path: String) throws {
         switch path.first {
-        case "/":
-            self.init(path)
         case "~":
             throw PathValidationError.startsWithTilde(path)
         default:
-            throw PathValidationError.invalidAbsolutePath(path)
+            guard (path as NSString).isAbsolutePath else {
```

Comment:
I am worried about the performance impact of this. Can we measure the difference or otherwise reimplement it Windows?

https://github.com/apple/swift-corelibs-foundation/blob/c5357f39dd15ae858dc72d15bbabe5435376db70/Foundation/NSPathUtilities.swift#L120-L126

Reply:
Do you have an easy/automated way to test the performance impact?

Reply:
You can generate Xcode project with SwiftPM perf tests enabled using: `Utilities/bootstrap --generate-xcodeproj --enable-perf-tests`
 
Then add a perf test here and run using the `PerformanceTest`:
You can write a test here: https://github.com/apple/swift-package-manager/tree/master/Tests/BasicPerformanceTests

See: https://github.com/apple/swift-package-manager/blob/master/Documentation/Development.md#running-the-performance-tests